### PR TITLE
fix(Slider): icon should be inactive when value is null

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
Currently, when the slider value is null, we have the first icon active ( when we use icons of course )
**What is the chosen solution to this problem?**
Add a if to do so

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/Slider/Slider.component.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ function isIconsAvailables(icons) {
  * @param {number} max - maximum value of the slider
  */
 export function getSelectedIconPosition(icons, value, min, max) {
+	if (value === null) {
+		return -1;
```
Comment (acateland) [inline packages/components/src/Slider/Slider.component.js]: why not throwing and forcing the client of this function to handle the error case cleanly ?

Comment (romainseb) [reply packages/components/src/Slider/Slider.component.js]: It's not an "error", it allow us to reset the slider to "no value" state.

File: packages/components/src/Slider/Slider.component.js [hunk 2]
```diff
@@ -26,6 +26,9 @@ function isIconsAvailables(icons) {
  * @param {number} max - maximum value of the slider
  */
 export function getSelectedIconPosition(icons, value, min, max) {
+	if (value === null) {
```
Comment (ncomont) [inline packages/components/src/Slider/Slider.component.js]: Why didn't you check for undefined values ?  With a simple `value == null` you could handle it at the same time.

File: packages/components/src/Slider/Slider.snap.test.js [hunk 3]
```diff
@@ -63,5 +63,14 @@ describe('Slider component', () => {
 			// then
 			expect(result).toBe(2);
 		});
+
+		it('should return -1 when value is null', () => {
```
Comment (ncomont) [inline packages/components/src/Slider/Slider.snap.test.js]: I think that this kind of tests has nothing to do in `snapshot` tests file. `FilterBar`, for example, has a `FilterBar.snapshot.test.js` for snapshot tests and a `FilterBar.test.js` for unit tests. Could you move it ?

**Pull request merged**